### PR TITLE
[plugin.video.videolina] 1.0.0

### PR DESCRIPTION
**Pull request by NextStepUnione**
### Description

Watch Videolina live streaming

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by NextStepUnione** [comment]
@enen92 is it eligible to be published now?

**Comment by Tadly** [comment]
Hi there.
We are kinda worried about the fanart you've included due to trademark etc.

Further, if included, the fanart should be more representing of your actual add-on.
If it's just a big red wall with the `V` in the middle, that's fine too.

Hope you understand :)

File: plugin.video.videolina/addon.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import xbmc
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+import logging
+from operator import itemgetter
```
Comment (Tadly) [inline plugin.video.videolina/addon.py]: `xbmcplugin`, `xbmcaddon`, `logging` and `itemgetter` aren't used, so you don't have to import them (unless I am missing something).
I'd be fine merging it but just so you know for next time ;)

**Comment by enen92** [comment]
@NextStepUnione I sort of agree with @Tadly. Its probably better not to have kodi's branding/trademark on third party plugins. At least so explicitly, to minimize confusion. Can you change the fanart please?
Also please have a look at those imports since you don't need them all (that one I'd already mentioned before).

**Pull request closed without merging**